src/aosm/azext_aosm/configuration_models/onboarding_nfd_base_input_config.py

```diff
@@ -26,6 +26,12 @@ class OnboardingNFDBaseInputConfig(OnboardingBaseInputConfig):
         },
     )
 
+    @property
+    def acr_manifest_name(self) -> str:
+        """Return the Storage account manifest name from the NFD name."""

```

Comment:
Doc string is wrong, it's not the storage account, and it's not using the NFD name